config.yaml

```
@@ -0,0 +1,10 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  auto-delete:
+    type: boolean
+    description: |
+        When a relation is removed, auto-delete ensures that any relevant databases
+        associated with the relation are also removed
+    default: true
```

I think it's better to change the default value to `false` to avoid the database being deleted when it's not wanted if some operator incorrectly removes the relation.

```suggestion
    default: false
```

Mykola change it on https://github.com/delgod/mongodb-k8s-operator/pull/13/files.